src/text_file_functions.cpp

```diff
@@ -223,7 +223,12 @@ QVector<server_type> AOApplication::read_serverlist_txt()
   while (!in.atEnd()) {
     QString line = in.readLine();
     server_type f_server;
-    QStringList line_contents = line.split(":");
+    QStringList line_contents;
+
+    if (!line.contains(','))
```

Comment:
nice

Reply:
What if the server name contains a comma?

Reply:
should use regex instead

Reply:
![image](https://user-images.githubusercontent.com/2956568/95910176-44ad5400-0d65-11eb-9617-9af7965fb755.png)
![image](https://user-images.githubusercontent.com/2956568/95910211-5131ac80-0d65-11eb-97d3-f2fd6c4d85db.png)